core/src/scss/components/alert/_alert.scss

```diff
@@ -7,51 +7,150 @@
 // of a status, or for displaying a validation message that
 // alerts someone of an important piece of information.
 //
-// .su-alert--success  - Successful Transaction Message
-// .su-alert--warning  - Warning Message
-// .su-alert--error    - Error Message
-// .su-alert--info     - Informative Message
+// .su-alert--success su-alert--text-light    - Successful Transaction Message
+// .su-alert--warning                         - Warning Message
+// .su-alert--error su-alert--text-light      - Error Message
+// .su-alert--info su-alert--text-light       - Informative Message
+// .su-alert--text-light                      - Lighter color text for darker backgrounds
```

Comment:
Are we including the alert variant with the big icon in this PR, or just the small icon version?

![Screen Shot 2020-09-01 at 10 43 55 AM](https://user-images.githubusercontent.com/42749717/91888202-be1f3600-ec40-11ea-85fa-002415f7a6a0.png)

Reply:
FYI looks like there are different padding values in the container depending on whether the alert has a big Alert Header and an Alert Footer.

Reply:
Yeah, just the one variant for this release. I hope to come back to the other variants later.